kubernetes/README.md

```diff
@@ -1067,4 +1069,46 @@ http://192.168.64.62:30911
 <div align="right">
     <b><a href="#table-of-contents">↥ back to top</a></b>
 </div>
-<br/>
\ No newline at end of file
+<br/>
+
+### Fault Injection
+
+`FaultInjection` policy helps you to test your microservices against resiliency. Kuma provides 3 different types of failures that could be imitated in your environment: delays, aborts, and response bandwidth limits. 
+
+#### Adding Fault Injection Policy
+
+In the following demo, we will be adding one policy that encompasses all three types of failures. However, you may break this policy apart as you see fit and only use the ones that are necessary for testing your microservices. Run the following command:
+
+```bash
+$ cat <<EOF | kubectl apply -f - 
+apiVersion: kuma.io/v1alpha1
+kind: FaultInjection
+mesh: default
+metadata:
+  namespace: default
+  name: everything
+spec:
+    sources:
+        - match:
+            service: frontend.kuma-demo.svc:8080
```

Comment:
Also probably worth mentioning, `service` isn't a mandatory tag in contrast to some other policies. The same matching could be done using  `app: kuma-demo-backend` and `app: kuma-demo-frontend`, which is a bit more user-friendly

Reply:
I'd go with `service` for now. It's consistent with all other stuff in the demo